phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -708,7 +705,7 @@ protected static String getOrganizationId() {
 
     private static long timestamp;
 
-    public static long nextTimestamp() {
+    public static synchronized long nextTimestamp() {
```

Comment:
Looks like this is recent addition right? We could include `deletePriorMetaData()` too?

Reply:
Or maybe just synchronizing `deletePriorMetaData()` could be enough as it would cover `deletePriorTables()`, `nextTimestamp()` and `deletePriorSchemas()`?

Reply:
Those are only used when running the tests on a real cluster, which feature is broken in many ways.
Almost no-one uses it, and it doesn't affect the maven test suite.

And by the pherf tests, which really should be fixed :)
It doesn't hurt to synchronize it.